contracts/bridge/BridgeMultiSig.sol

```diff
@@ -0,0 +1,74 @@
+pragma solidity ^0.4.24;
+
+import "../externals/openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "../externals/openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
+import "../externals/openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../externals/openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract BridgeMultiSig is Ownable {
+    mapping (address => bool) public signers;
+    mapping (bytes32 => mapping (address => bool)) public signedTxs; // <sha3(type, args, nonce), <singer, vote>>
+    mapping (bytes32 => uint64) public signedTxsCounts; // <sha3(type, args, nonce)>
+    mapping (bytes32 => uint64) public committedTxs; // <sha3(type, nonce)>
+    mapping (uint64 => uint64) public signerThresholds; // <tx type>
+    mapping (uint64 => uint64) public configurationNonces; // <tx type, nonce>
+
+    enum TransactionType {
```

Comment:
`TransactionType` makes me confused since Klaytn already has used the term with other meaning. 
 How about using another term such as `Command`, `MessageType`, or `AnchoringType`.

Reply:
VoteType :)